packages/elements/src/internals/machines/sign-in/verification.machine.ts

```diff
@@ -92,49 +92,66 @@ const SignInVerificationMachine = setup({
         return;
       }
 
-      if (
-        !clerk.client.signIn.supportedFirstFactors.every(factor =>
-          context.registeredStrategies.has(factor.strategy as unknown as SignInFactor),
-        )
-      ) {
-        console.warn(
-          `Clerk: Your instance is configured to support these strategies: ${clerk.client.signIn.supportedFirstFactors
-            .map((factor: any) => factor.strategy)
-            .join(', ')}, but the rendered strategies are: ${Array.from(context.registeredStrategies).join(
-            ', ',
-          )}. Before deploying your app, make sure to render a <Strategy> component for each supported strategy. For more information, visit the documentation: https://clerk.com/docs/elements/reference/sign-in#strategy`,
-        );
-      }
+      // Only show these warnings in development!
```

Comment:
I think it makes sense to show all of them only during development, or what do y'all think?

Reply:
@LekoArts I'm torn on this one as a prod config can differ from dev.